pkg/adapter/metrics.go

```diff
@@ -19,11 +19,21 @@ import (
 	"time"
 )
 
+// Metric kinds supported by mixer.
 const (
-	// GAUGE is used to record instantaneous (non-cumulative) measurement
-	GAUGE Kind = iota
-	// COUNTER is used to record increasing cumulative values.
-	COUNTER
+	Gauge   Kind = iota // records instantaneous (non-cumulative) measurements
+	Counter             // records increasing cumulative values
+)
+
+// Label kinds supported by mixer.
+const (
```

Comment:
I added my justification in the comments above.

I'm not convinced there is value in supporting types that aren't supported by the language, at the moment.  Even IPAddr and URL seem a bit overkill for our initial integration pieces. Prometheus supports string labels only. Statsd doesn't even support labels. Stackdriver only supports String, Bool, and Int64.

I don't think we want to include that enum directly. We certainly haven't been passing descriptors down directly in the rest of the code. And I think we want to insulate backend adapters from any API changes, as best we can.